anomalib/models/components/freia/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Framework for Easily Invertible Architectures.
+
+Module to construct invertible networks with pytorch, based on a graph
+structure of operations.
```

Comment:
Could you insert a link to the github page of the project?
Without it it is not convenient to search where it was taken.

Reply:
Done.